### PR TITLE
CLI accepts params replacements

### PR DESCRIPTION
**Pull request by dawidkeshop**
There is no way to have a profit from this code currently, but as for now it can accept, validate and parse parameters from command line.

The syntax is: `configure-eda -i "C:/Temp" -p evo-host=test.eshopworld.com -p sample=value`

### CODE REVIEW

File: src/Tests/Platform.Eda.Cli.Tests/Commands/ConfigureEda/OptionsValidation/ReplacementParamsDictionaryAttributeTests.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        [InlineData("=def")]
+        [InlineData("abc")]
+        public void ForInvalidParam_ReturnsError(string invalidValue)
+        {
+            var result = new ReplacementParamsDictionaryAttribute().GetValidationResult(invalidValue, new ValidationContext(new object()));
+
+            result.Should().NotBe(ValidationResult.Success);
+        }
+    }
+}
```
Comment (tabman83) [inline src/Tests/Platform.Eda.Cli.Tests/Commands/ConfigureEda/OptionsValidation/ReplacementParamsDictionaryAttributeTests.cs:30]: What happens if we have blanks like `abc =def` or commas `evo-domain="test.domain.com"`

Comment (dawidkeshop) [reply src/Tests/Platform.Eda.Cli.Tests/Commands/ConfigureEda/OptionsValidation/ReplacementParamsDictionaryAttributeTests.cs]: The first one is invalid, the second one is valid. I've updated regex and tests.

**Review by tabman83 — APPROVED** [review]
Happy to approve with just a suggestion. Should we just not trim the spaces? Not saying that it is the correct behavior but let's think about that.

**Comment by dawidkeshop** [comment]
> 
> 
> Happy to approve with just a suggestion. Should we just not trim the spaces? Not saying that it is the correct behavior but let's think about that.

Actually it won't be an issue. If CLI is executed with arguments containing spaces, like that:
 `configure-eda -i "C:/Temp" -p evo-host= test.eshopworld.com` it will just fail due to unrecognized syntax.

**Pull request merged**